docs/reference/esql/esql-get-started.asciidoc

```diff
@@ -1,34 +1,434 @@
+`|`. Processing commands change an input table by adding, removing, or changing
+rows and columns. Processing commands can perform filtering, projection,
+aggregation, and more.
+
+image::images/esql/esql-limit.png[A processing command changing an input table,align="center",width="60%"]
+
+For example, you can use the <<esql-limit>> command to limit the number of rows
+that are returned, up to a maximum of 10,000 rows:
+
+[source,esql]
+----
+FROM sample_data
+| LIMIT 3
+----
+
+[TIP]
+====
+For readability, you can put each command on a separate line. However, you don't
+have to. The following query is identical to the previous one:
+
+[source,esql]
+----
+FROM sample_data | LIMIT 3
+----
+====
+
+[discrete]
+[[esql-getting-started-sort]]
+==== Sort a table
+
+image::images/esql/esql-sort.png[A processing command sorting an input table,align="center",width="60%"]
+
+Another processing command is the <<esql-sort>> command. By default, the rows
+returned by `FROM` don't have a defined sort order. Use the `SORT` command to
+sort rows on one or more columns:
+
+[source,esql]
+----
+FROM sample_data
+| SORT @timestamp DESC
+----
+
+[discrete]
+[[esql-getting-started-where]]
+==== Query the data
+
+Use the <<esql-where>> command to query the data. For example, to find all
+events with a duration longer than 5ms:
 
-image::images/esql/processing-command.svg[A processing command changing an input table,align="center"]
+[source,esql]
+----
+FROM sample_data
+| WHERE event.duration > 5000000
+----
+
+`WHERE` supports several <<esql-operators,operators>>. For example, you can use <<esql-like-operator>> to run a wildcard query against the `message` column:
+
+[source,esql]
+----
+FROM sample_data
+| WHERE message LIKE "Connected*"
+----
+
+[discrete]
+[[esql-getting-started-more-commands]]
+==== More processing commands
+
+There are many other processing commands, like <<esql-keep>> and <<esql-drop>>
+to keep or drop columns, <<esql-enrich>> to enrich a table with data from
+indices in {es}, and <<esql-dissect>> and <<esql-grok>> to process data. Refer
+to <<esql-processing-commands>> for an overview of all processing commands.
+
+[discrete]
+[[esql-getting-started-chaining]]
+=== Chain processing commands
 
 You can chain processing commands, separated by a pipe character: `|`. Each
-processing command works on the output table of the previous command.
+processing command works on the output table of the previous command. The result
+of a query is the table produced by the final processing command.
+
+image::images/esql/esql-sort-limit.png[Processing commands can be chained,align="center"]
+
+The following example first sorts the table on `@timestamp`, and next limits the
+result set to 3 rows:
+
+[source,esql]
+----
+FROM sample_data
+| SORT @timestamp DESC
+| LIMIT 3
+----
+
+NOTE: The order of processing commands is important. First limiting the result
+set to 3 rows before sorting those 3 rows would most likely return 3 different
```

Comment:
Not sure what you meant here: is that the result of `sort | limit` is different than the one of `limit | sort`? If so the statement " First limiting the result
set to 3 rows before sorting those 3 rows would most likely return 3 different rows" would be better off like " First limiting the result set to 3 rows before sorting those 3 rows would most likely return a result that is different than the one where the sorting comes before the limit".

Reply:
Yes, that's what I was trying to say here. Thanks! I've changed it into your suggestion.